packages/flutter_tools/templates/app/linux.tmpl/window_configuration.cc.tmpl

```diff
@@ -3,3 +3,4 @@
 const char *kFlutterWindowTitle = "{{projectName}}";
 const unsigned int kFlutterWindowWidth = 1280;
 const unsigned int kFlutterWindowHeight = 720;
+const bool kUseHeaderBar = true;
```

Comment:
Do we expect 99% of people to care? If not, I don't think we need to make this configurable in the template; title and initial size were only done because it was so common for people to want to set them. In general, if someone wants to configure the template, the right way to do it is for them to just change the code once it's generated.

(In fact, we should remove window_configuration.cc at this point, and fold it into the files that set it, since we aren't expecting any more nuke-the-whole-folder breaking changes.)

Reply:
I would say most people don't care, and the ones that do will just modify the generated code. So I'll happily remove the boolean.